guides/common/modules/proc_creating-a-custom-ssl-certificate.adoc

```diff
@@ -88,6 +88,18 @@ You can also set a wildcard value `*`.
 If you set a wildcard value, you must add the `-t {certs-proxy-context}` option when you use the `katello-certs-check` command.
 <2> Under `[alt_names]`, include the FQDN of the load balancer as `DNS.1` and the FQDN of {SmartProxyServer} as `DNS.2`.
 endif::[]
++
+For more information about the `[ v3_req ]` parameters and their purpose, see link:https://www.rfc-editor.org/rfc/rfc5280[RFC 5280: Internet X.509 Public Key Infrastructure Certificate and Certificate Revocation List (CRL) Profile].
++
+[NOTE]
+====
+If you are using the TLS{nbsp}1.2 protocol, add also the `keyEncipherment` key usage bit:
```

Comment:
We have not added full TLS 1.3 support, and our default certificate setup sets:

```
keyUsage = digitalSignature, keyEncipherment
```

Therefore, I would prefer if we drop this section and just document line 52 and 77 as the above. I think you can then safely take this out of draft.

Reply:
Thanks! I reverted the commit that added this.